src/content/docs/change-tracking/ci-cd/change-tracking-jenkins.mdx

```diff
@@ -0,0 +1,287 @@
+
+    <img
+      title="Screenshot showing how to find Deployment Notifications"
+      alt="Screenshot showing how to find Deployment Notifications"
+      src={trackingJenkinsDeployNotification}
+    />
+4. Complete the following fields:
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            Field
+          </th>
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+        <tbody>
+        <tr>
+          <td>
+            API Key
+          </td>
+          <td>
+            API Credentials to conduct the deployment action in New Relic. Credentials can be created by creating new credentials in Jenkins using a username and password. Specify any identifier for the username, and the API key for the password.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            applicationID
+          </td>
+          <td>
+            This is only used for calls to the REST API. Do not include this if you're creating a call to the GraphQL endpoint.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Description
+          </td>
+          <td>
+            Insert a description of what you are tracking.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Revision
+          </td>
+          <td>
+            Revision for the deployment.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Changelog
+          </td>
+          <td>
+            Reference to the list of changes incurred in the deployment.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Commit
+          </td>
+          <td>
+            A hash to reference the commit id.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Deeplink
+          </td>
+          <td>
+            A deep link to either the Jenkins job or any other reference to the deployment.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            DeploymentId
+          </td>
+          <td>
+            An identifier to reference the deployment ID.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            EntityGuid
+          </td>
+          <td>
+            The application entity in reference to the deployment (see [More about the application GUID](#guid)).
+          </td>
+        </tr>
+        <tr>
+          <td>
+            DeploymentType
+          </td>
+          <td>
+            The type of deployment that is being conducted.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            GroupId
+          </td>
+          <td>
+            An identifier to reference a cluster of deployments.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            Timestamp
+          </td>
+          <td>
+            Optional timestamp in Unix Epoch long format. If not provided, then New Relic will assume the time.
```

Comment:
New Relic will assume the time in which format?

Reply:
@a-sassman Here's the final answer: Optional timestamp in Unix Epoch long format. If not provided, then we will set it to the time we received the request, which will be the number of milliseconds since the Unix Epoch.